### DISCUSSION TITLE
Advanced Module - moving blocks

### DISCUSSION THREAD

**Discussion by garryatbrighton** [YetiForceCompany/YetiForceCRM#15269] [How to]
How to move blocks to left column except line items in advanced module? (edit view)

**Comment by rskrzypczak** [comment]
Check this: https://github.com/YetiForceCompany/YetiForceCRM/discussions/14814

**Comment by JohnMaverick** [comment]
for me it not works in latest stable app

**Reply by mariuszkrzaczkowski** [reply]
tested on gitstable and working fine
```php
/** Is divided layout style on edit view in modules with products */
public static $INVENTORY_EDIT_VIEW_LAYOUT = false;
```
